tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConformanceTests.cs

```diff
@@ -111,25 +112,21 @@ protected override void SetupConnectionInformationIsDelayValidated()
         throw new SkipTestException("Need to skip this test until https://github.com/npgsql/efcore.pg/issues/2891 is fixed.");
```

Comment:
We should be able to remove this now that we have `builder.EnableServiceProviderCaching(false);` in ConfigureDbContextOptionsBuilderForTesting.

Reply:
That works, but I had to add a similar thing for Enrich instead since it doesn't care about the connection string.